### DISCUSSION TITLE
Option to disable Recommendations in "For You" Tab

### DISCUSSION THREAD

**Discussion by dragos-dijmarescu** [community/community#55840] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Seems like Github is turning into Linkedin for some reason. I don't believe that for me, as well as for other users, the Recommendations part in the For You section is all that useful. I would like to have the option to change this feature or customize the start page like having code snippets from my projects or my latest commit appear in place of the "For You" - Recommendations section. 
EDIT: This used to have a toggle button in Settings as mentioned in Github post https://github.com/TeamNewPipe/NewPipe/issues/5173. I wonder why would they remove it ?

**Comment by ArjixWasTaken** [comment]
> Disable the "For you" tab

Nobody is forcing you to use it, you can use the "Following" tab.

> I would like to have github default to my repository page.

Have you ever heard of browser bookmarks?
I think they would be really useful to you, instead of asking GitHub to make such a change.

> https://github.com/TeamNewPipe/NewPipe/issues/5173

What does that issue have to do with this?

**Reply by dragos-dijmarescu** [reply]
It used to be a toggle in Settings, but they have removed it.

**Reply by dragos-dijmarescu** [reply]
Neither of your comments are constructive or useful. You guys should go outside and get some sunlight, clear your minds a little 😉

**Reply by ArjixWasTaken** [reply]
> I am not that keen in using Github anyway. They can do whatever with their linkedin number 2 💩

I am not sure what you meant by that, but if you think LinkedIn is better than GitHub, then I would be glad if you never used GitHub again as well.

**Reply by dragos-dijmarescu** [reply]
Your opinion does not matter to me at all.

**Comment by beta-tester** [comment]
for me (at the moment) i could disable this annoying "Recommend for you" by open "Filter" option (at the top right of the view), scrolled down to the entry "Recommendations", disabling that option and saved my choice.

![image](https://github.com/community/community/assets/4750719/2a840ecc-f04e-44c5-91bd-b0126419ca41)

**Reply by BigShuff** [reply]
Thanks for that damn things drive me nuts.
